### PR TITLE
Parameterize RCEMIPII sst distribution

### PR DESCRIPTION
**Pull request by cmschmitt519**
## Purpose 

Add parameters for SST distribution
Modify Earth rotation and ocean albedo for RCEMIP
Keep short run configurations in model configs

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: src/parameters/Parameters.jl [hunk 1]
```diff
@@ -44,6 +48,13 @@ Base.@kwdef struct TurbulenceConvectionParameters{FT} <: ATCP
     max_area_limiter_power::FT
 end
 
+Base.@kwdef struct SurfaceTemperatureParameters{FT} <: ASTP
+    SST_mean::FT
+    SST_delta::FT
+    SST_wavelength::FT
+    SST_wavelength_latitude::FT
+end
+
```
Comment (szy21) [inline src/parameters/Parameters.jl:57]: @nefrathenrici Is it better to group parameters in a struct like this or add them to ClimaAtmosParameters?

Comment (cmschmitt519) [reply src/parameters/Parameters.jl:57]: I grouped them like this so they would be broadcasted correctly into surface_state_to_conditions. ClimaAtmosParameters never gets passed into this function, where surface_temperature is called.

Comment (szy21) [reply src/parameters/Parameters.jl:57]: That makes sense, thanks!

File: src/surface_conditions/surface_conditions.jl [hunk 2]
```diff
@@ -332,28 +340,33 @@ end
...
 function surface_temperature(
     ::RCEMIPIISST,
     coordinates::Union{Geometry.XZPoint, Geometry.XYZPoint},
+    surface_temp_params,
 )
     (; x) = coordinates
+    (; SST_mean, SST_delta, SST_wavelength) = surface_temp_params
     FT = eltype(x)
-    T = FT(300) + FT(1.25) / 2 * cos(2 * FT(pi) * x / 6000)
+    T = SST_mean + SST_delta / 2 * cos(2 * FT(pi) * x / SST_wavelength)
```
Comment (szy21) [inline src/surface_conditions/surface_conditions.jl:361]:
```suggestion
    T = SST_mean - SST_delta / 2 * cos(2 * FT(pi) * x / SST_wavelength)
```
(It's a minus sign in Eq. 1 in Wing et al. 2023?)

File: toml/rcemipii_diagnostic_edmfx_0M.toml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+value = 0
+
+[angular_velocity_planet_rotation]
+value = 0
+
+[ocean_surface_albedo]
+value = 0.07
+
+[mean_sea_level_pressure]
+value = 101480
```
Comment (szy21) [inline toml/rcemipii_diagnostic_edmfx_0M.toml:32]: Could you add an empty line at the end of this file?

**Review by szy21 — APPROVED** [review]
Thank you!

**Pull request merged**